subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractContinuousIntegrationTest.groovy

```diff
@@ -147,7 +148,10 @@ ${result.error}
         executer.withStdinPipe()
             .withTasks(tasks)
             .withForceInteractive(true)
+            .withArgument("--info")
```

Comment:
❓ Can you add a comment to the test why you add the `--info` logging and `console=plain` here? I am not sure this is a good idea since this change would affect all continuous build integration tests.

Reply:
Unintentional left-overs, will remove it.